### PR TITLE
mf1933114

### PR DESCRIPTION
**Pull request by ZaiXiaochen**
|ClassName|Row|ChangedOrNot|
|---|---|---|
|Collections|222|unchanged|
|Class|166|changed|
|String|178|unchanged| 
|LinkedList|74 | unchanged|
|HashMap|66| unchanged|

### CODE REVIEW

File: MF1933114/java/Class_getDeclaringClass.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+ *  @apiSignature: java.lang.Class$public Class getDeclaringClass()
+ *  @description: Test java api java.lang.Class$public Class getDeclaringClass()
+ *  @Map: none
+ */
+import java.lang.reflect.Method;
+
+public class Class_getDeclaringClass{
+    /**
+	*input:1
+	*m Method$m=String.class.getMethods()[0]
```
Comment (idiomaticrefactoring) [inline MF1933114/java/Class_getDeclaringClass.java]: 变量名定义

File: MF1933114/java/Collections_max_Collection_Comparator.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+ *  @Map: ContiguousArray$ func max(by areInIncreasingOrder: (Element, Element)->Bool )->Element
+ *  @Map: Array$ func max(by areInIncreasingOrder: (Element, Element)->Bool )->Element
+ */
+import java.util.*;
+
+public class Collections_max_Collection_Comparator{
+	public static void max0(){
+	/**
+	*input:2
+	*coll List<Integer>$coll=[1,12,3,7]
```
Comment (idiomaticrefactoring) [inline MF1933114/java/Collections_max_Collection_Comparator.java]: coll格式错误后面同样问题

**Review by idiomaticrefactoring — CHANGES_REQUESTED** [review]
good

**Review by ZaiXiaochen — COMMENTED** [review]
changed some code files

**Pull request closed without merging**